Function Development/Data Tracking/.env.example

```
@@ -0,0 +1,8 @@
+# 复制此文件为 .env，填写真实值，并确保 .env 已加入 .gitignore
+# NEVER commit .env to version control
+
+DB_HOST=106.14.119.61
+DB_PORT=5432
+DB_USER=workbench
+DB_PASS=workbench123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASS key should go before the DB_PORT key

(UnorderedKey)